### PR TITLE
Badge dropdown

### PR DESCRIPTION
**Pull request by iviondayjr**
Badge select / dropdown implemented
minor ui:
- button styling
- textarea code font
- flexbox darkened
- cursor pointer on radio button hover

{F7FE41FE-7263-484B-BCFA-57688EDD5F81}

### CODE REVIEW

**Comment by Meltem-Karaagac** [comment]
It looks great.

**Review by Meltem-Karaagac — APPROVED** [review]
Great.

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -343,6 +351,7 @@ class App extends React.Component {
               className="links"
               id="example-links"
               name="links"
+              spellcheck="false"
```
Comment (rvrita) [inline client/src/components/App.jsx:354]: Nice

**Review by rvrita — APPROVED** [review]
This looks great, I love the small design changes too! Could you please add the other Allure badges too? Thank you!!! 🥳

**Review by choiwa — APPROVED** [review]
hope this json you will find it useful.
[
    {
      "name": "None",
      "value": ""
    },
    {
      "name": "Clean",
      "value": "pb=2020-03-sephora-clean-2019&"
    },
    {
      "name": "Clean+",
      "value": "pb=clean-planet-positive-badge-2021&"
    },
    {
      "name": "Allure 2018",
      "value": "pb=2020-03-allure-best-2018&"
    },
    {
      "name": "Allure 2019",
      "value": "pb=2020-03-allure-best-2019&"
    },
    {
      "name": "Allure 2020",
      "value": "pb=allure-best-2020&"
    },
    {
      "name": "Allure 2021",
      "value": "pb=allure-best-2021-badge&"
    },
    {
      "name": "Value",
      "value": "pb=2020-03-sephora-value-2019&"
    }
  ]

and the css styles needs "grid-generator" added for Orion.

**Pull request merged**